composer.json

```diff
@@ -47,7 +47,7 @@
         "nelmio/api-doc-bundle": "<2.4",
         "sonata-project/block-bundle": "<3.11",
         "sonata-project/doctrine-orm-admin-bundle": "<3.0",
-        "sonata-project/google-authenticator": "<1.0",
+        "sonata-project/google-authenticator": "<2.1",
```

Comment:
do we still need this conflict section entry?

Reply:
true ;) removed